website/migrations/20200325003307_initial-tables.js

```diff
@@ -0,0 +1,178 @@
+const path = require('path');
```

Comment:
Should this file be git tracked?

Reply:
Lol , it's [`path`](https://nodejs.org/api/path.html), not `./path` 😄

Reply:
I mean the file... not this specific line :D

Reply:
Aha, sorry.

You mean, should we have the migration that creates the tables in git? Sure, why not? It makes it pretty easy to setup a dev environment. Or a prod environment for that matter...